code.js

```diff
@@ -38,8 +38,31 @@ app.get('/status', (request, response) => {
   }
 });
 
+function getSprintStart() {
+  // Get the start of sprint. Messy but probably works okay for now
```

Comment:
Normally, won't the current milestone have a date range you can use?

Reply:
Then fallback to this if it's not set

Reply:
I wanted to avoid two calls to GitHub -- first to retrieve sprint timetable and second to retrieve this data. But you are right, it's probably cleaner to do that :)